spec/peep_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'peep'
+
+describe Peep do
+  describe '.all' do
+    it 'returns all peeps' do
+      peeps = Peep.all
+
+      expect(peeps.count).to eq 2
+      expect(peeps.first['peep']).to eq '¡Día de partido! ¡Vamos Real, hasta el final!'
+      expect(peeps[1]['peep']).to eq 'El bicho scores a hattrick!'
+    end
+  end
+end
```

Comment:
It would be clearer to make the set up part of this file.  You could, for example, have a before block which inserts entries into the db.  Then do `expect(Peep.all).to match_array ["el coche es antiguo", "donde esta Steve McManaman"]`.  The great thing about `match_array` is that it doesn't matter what order the items are in.

Reply:
Awesome matcher! I did not know it existed!

What about this?
```
describe Peep do
  before(:all) do
    @peeps = []
    peep_objects = Peep.all
    
    peep_objects.each { |peep| @peeps << peep.peep }
  end

  describe '.all' do
    it 'returns all peeps' do
      expect(@peeps).to match_array ['El bicho scores a hattrick!', '¡Día de partido! ¡Vamos Real, hasta el final!']
    end
  end
end
```